pipelines/candlepin/02-install.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Setup git repo
+  become: True
+  hosts:
+   - "{{ forklift_name }}"
+  vars_files:
+    - ../vars/forklift_candlepin.yml
+  tasks:
+    - name: Install podman-docker
+      package:
+        name: podman-docker
+        state: installed
+
+    - name: Start podman
+      service:
+        name: podman
+        state: started
+
+    - name: Install git
+      package:
+        name: git
+        state: installed
+
+    - name: Enable ruby 2.7 module
+      command: dnf module enable -y ruby:2.7
+
+    - name: Install Ruby
+      package:
+        name:
+          - ruby
+          - ruby-devel
+          - rubygem-bundler
+          - gcc-c++
+          - make
+          - redhat-rpm-config
+        state: installed
```

Comment:
given you're planning to re-use this for pulpcore, I think this could go to a `beaker` role?

Reply:
Fair.

Reply:
Done - let me know what you think.

Reply:
one comment, but I like!